src/static/js/pad.js

```diff
@@ -656,6 +664,7 @@ const pad = {
     pad.determineChatVisibility(isConnected && !isInitialConnect);
     pad.determineChatAndUsersVisibility(isConnected && !isInitialConnect);
     pad.determineAuthorshipColorsVisibility();
+    pad.determineFocus();
```

Comment:
Seems unnecessary to have a separate function for something so trivial.

Reply:
I mean there's no need to create a new `determineFocus` method:

```suggestion
    if (padcookie.getPref('focusOnEditor')) padeditor.ace.focus();
```